### ISSUE TITLE
"Generating a path" has a issue

### ISSUE THREAD

**Issue by Monobesatoshi** [SebLague/Path-Creator#128]
I read user guide and tried to generate a path on script, but it didn't work.  I added the PathCreation namespace and created a public PathCreator. my unity version is 2022 1.11 . Error came out like below.
![スクリーンショット 2022-11-02 180754](https://user-images.githubusercontent.com/83193595/199448448-bbca78d1-7729-4e53-b830-d97d829cbb28.png)
![スクリーンショット 2022-11-02 180845](https://user-images.githubusercontent.com/83193595/199448475-0dd782ae-eed1-4589-be6a-3af49380eb6d.png)
![スクリーンショット 2022-11-02 180934](https://user-images.githubusercontent.com/83193595/199448495-18153e6d-98aa-4233-889a-259e1d29b0f4.png)

**Comment by eric29** [comment]
I agree, same problem.  VertexPath function isn't working and don't know enough about the code to understand why.

It's really too bad the creator isn't looking at these. It seems to be a dead project that has so much potential and use.
